### PR TITLE
Update README (spelling, phrasing, formatting)

### PR DESCRIPTION
**Pull request by Zearin**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -118,92 +119,100 @@ The subtitle of the notification.
...
+Sound names are listed in Sound Preferences. 
+
+Use the special `NAME` “default” for the default notification sound.
 
 -------------------------------------------------------------------------------
 
 `-reply` **[10.9+ only]**
 
-The notification will be displayed as a reply type alert.
+Displayed the notification as a reply type alert.
```
Comment (julienXX) [inline README.markdown]: Display instead of Displayed

File: README.markdown [hunk 2]
```diff
@@ -118,92 +119,100 @@ The subtitle of the notification.
...
 
 -------------------------------------------------------------------------------
 
 `-actions VALUE1,VALUE2,"VALUE 3"` **[10.9+ only]**
 
-The notification actions avalaible.
+Use notification actions avalaible.
```
Comment (julienXX) [inline README.markdown]: `The list of actions available for the notification` maybe?

File: README.markdown [hunk 3]
```diff
@@ -1,27 +1,27 @@
 # terminal-notifier
 
-terminal-notifier is a command-line tool to send Mac OS X User Notifications,
-which are available in Mac OS X 10.8 and higher.
+terminal-notifier is a command-line tool to send OS X User Notifications,
```
Comment (julienXX) [inline README.markdown]: Maybe it's time to change for macOS which is the current name.

File: README.markdown [hunk 4]
```diff
@@ -212,45 +221,43 @@ Examples are:
 
 `-sender ID`
 
-Specifying this will make it appear as if the notification was send by that
-application instead, including using its icon.
+Fakes the sender application of the notification.  This uses the specified
```
Comment (julienXX) [inline README.markdown]: Too many spaces before `This` I think.

File: README.markdown (hunk 2 above)
Comment (MrMarc) [inline README.markdown]: _available_ is spelt incorrectly ...

File: README.markdown (hunk 3 above)
Comment (Zearin) [inline README.markdown]: Yeah, sounds good to me.  (I was on the fence about that one when I was editing anyway. )

**Comment by julienXX** [comment]
Thanks @Zearin !

**Pull request merged**